src/Microsoft.AspNetCore.Authentication.AzureADB2C.UI/AzureAdB2CDefaults.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.using Microsoft.AspNetCore.Authorization;
+
+
+using Microsoft.AspNetCore.Authentication;
+
+namespace Microsoft.AspNetCore.Authentication.AzureADB2C.UI
+{
+    /// <summary>
+    /// Constants for different Azure Active Directory B2C authentication components.
+    /// </summary>
+    public class AzureADB2CDefaults
+    {
+        /// <summary>
+        /// The key for the policy used in <see cref="AuthenticationProperties"/>.
+        /// </summary>
+        public const string PolicyKey = "Policy";
```

Comment:
Most of our 'XyzDefaults' classes use `static readonly`, though oddly https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerDefaults.cs doesn't. Unless these are actual universal constants (say, per some spec), I would use `static readonly`.